values/cloudnative-pg/cloudnative-pg.gotmpl

```diff
@@ -0,0 +1,9 @@
+{{- $v := .Values }}
+{{- $es := $v.apps | get "cloudnative-pg" }}
+
+limits:
+  cpu: 100m
+  memory: 200Mi
+requests:
+  cpu: 100m
+  memory: 100Mi
```

Comment:
Hi there,

Great work so far.
One thing - nowadays we try to avoid baking resource quotas into the templates, so instead of this you can replace with:
`resources: {{- $es.resources | toYaml | nindent 2 }}`

Then you can extend the values-schema.yaml with something like:
```
      cloudnative-pg:
        additionalProperties: false
        properties:
          _rawValues:
            $ref: '#/definitions/rawValues'
          enabled:
            type: boolean
            default: false
          resources:
            $ref: '#/definitions/resources'
```

then in defaults.yaml:
```
          cloudnative-pg:
            enabled: false
            resources:
              requests:
                cpu: 100m
                memory: 100Mi
              limits:
                cpu: 100m
                memory: 200Mi
```